conf/routes

```diff
@@ -111,5 +102,9 @@ POST   /admin/v1/backfill/:name/start    @controllers.BackfillController.start(n
 POST   /admin/v1/users          @controllers.UserManagementController.createUser
 DELETE /admin/v1/users          @controllers.UserManagementController.deleteUser(email: java.lang.String ?= null)
 
+# Worker - Health Data
+POST /worker/v2/healthdata @controllers.HealthDataController.createRecord
```

Comment:
We should be able to add a worker role now and enforce that in the controller, if you don't want to use admin.